src/resources/trainers/controller.js

```diff
@@ -14,6 +18,49 @@ async function getTrainers(req, res) {
     }
 }
 
+async function getOneTrainer(req, res) { 
+
+    try { 
+
+        const trainerById = await prisma.trainer.findUnique({
+            where : { 
+                id : parseInt(req.params.id)
+            },
+            include : { 
+                classes : true
+            }
+        })
+
+        res.json(trainerById)
+    } catch(error) { 
+        res.status(500).json({ error : error.message })
+    }
+
+}
+
+async function createTrainer(req, res) { 
+
+    const trainerDetails = req.body
+    console.log(" new trainer: ", trainerDetails)
```

Comment:
Might want to remove this console log to keep the output cleaner now that you've finished the function :)

Reply:
that's a good shout, will remove the console.log("PLEASE WORK") for this as well all the other resources that are working